build.gradle

```diff
@@ -10,7 +10,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'junit:junit:4.12'
+    testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.7.0'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
```

Comment:
Maybe you could upgrade this one to 5.7.0 too.

Reply:
I will update it in the following card